hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/misc/Pojo.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.misc;
```

Comment:
`misc` package contains tests to which this class is completely unrelated. Can you find a better place for it? We have a few `model` packages in tests.

Reply:
Done.

Reply:
still not a good place. `com.hazelcast.jet.sql.impl.schema` contains tests related to SQL schemas, `Pojo` is not used in them.